src/components/RenderGroceryById/RenderGroceryById.js

```diff
@@ -9,14 +9,12 @@ import shortid from "shortid";
 
 class RenderGroceryById extends Component {
   render() {
+    let id = parseInt(this.props.match.params.id, 10);
```

Comment:
can we move some of data, that we got from parents into a constructor.

something like this:
```
constructor(props) {
        super(props);
        this.state = { id: ... }
```